controllers/datadogagent/utils.go

```diff
@@ -825,6 +825,23 @@ func getVolumesForAgent(dda *datadoghqv1alpha1.DatadogAgent) []corev1.Volume {
 		volumes = append(volumes, volume)
 	}
 
+	// Dogstatsd volume
+	if datadoghqv1alpha1.BoolValue(dda.Spec.Agent.Config.Dogstatsd.UseDogStatsDSocketVolume) {
+		volumeType := corev1.HostPathDirectoryOrCreate
+		hostPath := *dda.Spec.Agent.Config.Dogstatsd.HostSocketPath
+
+		dsdsockerVolume := corev1.Volume{
+			Name: datadoghqv1alpha1.DogstatsdSockerVolumeName,
```

Comment:
```suggestion
			Name: datadoghqv1alpha1.DogstatsdSocketVolumeName,
```

Typo to be fixed at the other places in the code.

Reply:
thanks for catching this typo